src/Native/System.Net.Http.Native/CMakeLists.txt

```diff
@@ -1,10 +1,9 @@
 project(System.Net.Http.Native)
 
-find_library(CURL NAMES curl)
-if(CURL STREQUAL CURL-NOTFOUND)
-    message(SEND_ERROR "!!! Cannot find libcurl and System.Net.Http.Native cannot build without it. Try installing libcurl4-openssl-dev (or the appropriate package for your platform) !!!")
-    return()
-endif()
+find_package(CURL)
```

Comment:
The changes to this file are not described in the PR or commit. What is the issue?